doc/source/serve/advanced-guides/performance.md

```diff
@@ -74,6 +74,12 @@ Ray Serve allows you to fine-tune the backoff behavior of the request router, wh
 - `RAY_SERVE_ROUTER_RETRY_BACKOFF_MULTIPLIER`: The multiplier applied to the backoff time after each retry. Default is `2`.
 - `RAY_SERVE_ROUTER_RETRY_MAX_BACKOFF_S`: The maximum backoff time (in seconds) between retries. Default is `0.5`.
 
+### Configure proxy routing preferences
```

Comment:
this section needs to cover how to control local node, local AZ routing for inter deployment communication beyond proxy for completeness.